code/game/objects/items/grenades/flashbang.dm

```diff
@@ -14,23 +14,34 @@
 	do_sparks(rand(5, 9), FALSE, src)
 	playsound(flashbang_turf, 'sound/weapons/flashbang.ogg', 100, TRUE, 8, 0.9)
 	new /obj/effect/dummy/lighting_obj (flashbang_turf, LIGHT_COLOR_WHITE, (flashbang_range + 2), 4, 2)
+	flashbang_mobs(flashbang_turf, flashbang_range)
 	for(var/mob/living/M in get_hearers_in_view(flashbang_range, flashbang_turf))
 		bang(get_turf(M), M)

```

Comment:
I suggest you to remove this two lines unless you intended to double the pain for those poor tools caught in its AoE.

Reply:
FUCK HOW DID I NOT NOTICE LMAO